torch/_inductor/ir.py

```diff
@@ -3413,9 +3414,21 @@ def __init__(
         self.kernel = kernel
 
     def codegen(self, wrapper):
-        wrapper.writeline(
-            f"{self.get_name()} = {self.kernel}({', '.join(self.codegen_args())})"
-        )
+        if config.mkldnn_param_caching:
+            if not wrapper.mkldnn_param_utils_gen:
+                wrapper.mkldnn_param_utils_gen = True
+                wrapper.write_mkldnn_param_gen_utils()
+            wrapper.writeline(f"{self.get_name()}_id = {random.getrandbits(32)}")
```

Comment:
Do we really need randomize ids here? Would this break the persistent code cache hashing?

Reply:
In my test env it always generate with same random id may be due to seed setting. Anyway, you are right we can use and ordered number instead of random. I've updated code accordingly.